packages/framework/src/Model/Payment/Payment.php

```diff
@@ -482,4 +482,22 @@ public function getUuid()
     {
         return $this->uuid;
     }
+
+    /**
+     * @return array
+     */
+    protected function getGatewayPayments()
```

Comment:
missing return typehint

Reply:
Ok. I thought that we do not use type hinting in entities, but it is applied only on entity fields, if I am correct, right ? 

fixed

Reply:
https://github.com/shopsys/shopsys/pull/3355/commits/9670bc675188f75080148f384f0d2698c1cdaf54